src/main/scala/it/pagopa/interop/apigateway/api/impl/package.scala

```diff
@@ -143,10 +144,8 @@ package object impl extends SprayJsonSupport with DefaultJsonProtocol {
       EService(eservice.id, eservice.name, version = descriptor.version, state = descriptor.state.toModel)
 
     private def flatAttributes(attribute: CatalogManagementApiAttribute): Set[UUID] = {
-      val flattenAttributes: Option[Seq[String]] = for {
```

Comment:
originally we made a for comprehension because the ADT modeling of Attribute does not expose the implicit invariant that it MUST be either a single or a group attribute, not both.
Because of that, this was always reduced to None.

Reply:
🆗